### PR TITLE
Better logging messages...

### PR DESCRIPTION
**Pull request by teh3vil**
Third time lucky D:
Refined console output, made it more pokemon related. Added options to configure the console output and hide some more repetitive stuff for people like myself who like to stare at the console.log (It's like an old text RPG that we don't have to play!)

Also added pretty locations, they rinse your API calls but it looks pretty cool.

### CODE REVIEW

File: pgoapi/pgoapi.py [hunk 1]
```diff
@@ -547,11 +574,15 @@ def spin_nearest_fort(self):
                 self.disk_encounter_pokemon(nearest_fort['lure_info'])
 
         else:
-            self.log.info('No spinnable forts within proximity. Or server returned no map objects.')
+            if self.HEARTBEAT_DETAIL != "hidden":
+                self.log.info('[STANDING]\t- No PokeStops nearby, better carry on!')
+            self.log.debug('No forts within proximity, or server returned no map objects.')
```
Comment (rirze) [inline pgoapi/pgoapi.py]: Wait it says forts here....

**Comment by rirze** [comment]
I see you left the debug messages the same. Was that intentional?

File: pgoapi/pgoapi.py (hunk 1 above)
Comment (teh3vil) [inline pgoapi/pgoapi.py]: Woops, there's still some left to do it seems D:

**Comment by teh3vil** [comment]
Yeah that was intentional. I've also moved one or two lines from info into debug as they contained response/status information. Will have another crack at cleaning this up tomorrow

**Pull request closed without merging**